### ISSUE TITLE
semver.org 2.0.0 last FAQ uses 'semver' while master branch 2.0.0 uses 'SemVer'

### ISSUE THREAD

**Issue by hyiltiz** [mojombo/semver#288]
I think the master branch version should be labeled as 2.0.1 or something else; since no software should be different while having the same versions. Also, it should be great to directly host through github, or make a mirror sync for the web server with master branch here.

**Comment by jwdonahue** [comment]
> I think the master branch version should be labeled as 2.0.1 or something else;

Yes.  We should probably be somewhere in the 2.yy.zzz range by now.

Have you tried forking [semver/semver.org](https://github.com/semver/semver.org)?  That's repo that  produces the [https://semver.org/](https://semver.org/).

**Comment by jwdonahue** [comment]
@hyiltiz, this a very old issue. Do you intend to issue a PR to fix it?  If not, and if you don't have any further questions, please close this thread at your earliest possible convenience.

Thank you.

**Comment by hyiltiz** [comment]
I'd love to submit a PR but not sure which abbr. is preferred among `semver` `SemVer` `Semver` for `Semantic Versioning`. They all appear in the document in various places.
